packages/react-label/src/components/Label/useLabelStyles.ts

```diff
@@ -9,8 +9,7 @@ const useStyles = makeStyles({
     fontFamily: theme.global.type.fontFamilies.base,
     fontSize: theme.global.type.fontSizes.base[300],
     color: theme.alias.color.neutral.neutralForeground1,
-    display: 'inline-flex',
-    alignItems: 'flex-end',
+    display: 'block',
```

Comment:
Are you sure that `block` is correct here? Should it be `inline`? (which is the default for `<label>`, so could be removed if so)